packages/app/src/themes/dark_theme.tsx

```diff
@@ -1,5 +1,5 @@
 import { createTheme, SimpleThemeOptions } from '@backstage/theme';
-import { gold, common, myYellow as yellow } from "./colorTypes";
+import { blue, gray, green, gold, red, common } from "./colorTypes";
```

Comment:
```suggestion
import { blue, /* gray, green, gold, red, common */ } from "./colorTypes";
```

If these don't need to be used we can delete or comment them out.